src/components/Tutorial.vue

```diff
@@ -136,14 +136,14 @@ export default {
   },
   created() {
     this.store.reset();
+    this.store.setReferences(this.references);
```

Comment:
Doesn't seem like we need a default state in this component, since it's an immediate child of `Topic.vue` were we provide the store. I think we are fine here.